calabash-cucumber/lib/calabash-cucumber/gestures/device_agent.rb

```diff
@@ -116,6 +113,28 @@ def touch_hold(options)
         end
 
         # @!visibility private
+        def swipe(options)
+          dupped_options = options.dup
+
+          if dupped_options[:query].nil?
+            dupped_options[:query] = "*"
+          end
+
+          from_hash = query_for_coordinates(dupped_options)
+          from_point = from_hash[:coordinates]
+          element = from_hash[:view]
+
+          gesture_options = {
+            :duration => dupped_options[:duration]
+          }
+
+          direction = dupped_options[:direction]
+          force = dupped_options[:force]
+          to_point = Coordinates.end_point_for_swipe(direction, element, force)
+          device_agent.pan_between_coordinates(from_point, to_point, gesture_options)
+        end
```

Comment:
We have to return the view that was swiped as an array:

```
[from_hash[:view]]
```

I also think it makes sense to rename the `from_hash` variable to `hash`.